sections/futures/MarketInfoBox/MarketInfoBox.tsx

```diff
@@ -69,55 +69,56 @@ const MarketInfoBox: React.FC = () => {
 	}, [commitDeposit, marketInfo?.keeperDeposit]);
 
 	const getPotentialAvailableMargin = (
```

Comment:
I think this function needs to take the cross-margin fee into account. I found that the preview margin shows as greater than the actual final margin by the amount of the cross-margin fee.

<img width="375" alt="Screen Shot 2022-09-01 at 3 09 51 PM" src="https://user-images.githubusercontent.com/10401554/188014249-103cb77d-b2ac-43bc-a672-820a587fcd52.png">

<img width="373" alt="Screen Shot 2022-09-01 at 3 11 20 PM" src="https://user-images.githubusercontent.com/10401554/188014274-a1f3d399-ec44-4389-b243-81f87ae484e9.png">

Reply:
Transaction: https://blockscout.com/optimism/goerli/tx/0x092489a31a0b1971c74ca36fd0cea73d96e4bc659191b92d9030c4429a649aac

Reply:
Good spot, fixed.